CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This file documents all notable changes to Peggy.
 Unreleased
 ----------
 
+- [#299](https://github.com/peggyjs/peggy/issues/299) Add example grammar for a
```

Comment:
I'll fix this up after merge.  Apparently I didn't leave a template in the changelog after 2.0.1, because that was done i n such a rush.